enable/base.py

```diff
@@ -103,8 +104,8 @@ def str_to_font ( object, name, value ):
                     underline = underline)
     except:
         pass
-    raise TraitError, ( object, name, 'a font descriptor string',
-                        repr( value ) )
+    raise TraitError(( object, name, 'a font descriptor string',
+                        repr( value ) ))
```

Comment:
There's one set of parentheses too many here.

`raise TraitError(object, name, 'a font descriptor string', repr( value ))`